react/package-lock.json

```diff
@@ -13231,7 +13231,7 @@
     "react-router-dom": {
```

Comment:
Looks like this file is autogenerated, if so, it  should be added to .gitignore

Reply:
Again discussed and we have added this in the past as part of running sbt dist  - we agreed to add it

Reply:
According to npm documentation we should continue to commit this file : This file is intended to be committed into source repositories : https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json